cmd/oras/copy.go

```diff
@@ -91,21 +90,9 @@ func runCopy(opts copyOptions) error {
 	}
 	var desc ocispec.Descriptor
 	if opts.rescursive {
-		desc, err = oras.ExtendedCopy(ctx,
-			src,
-			srcRef.ReferenceOrDefault(),
-			dst,
-			dstRef.ReferenceOrDefault(),
-			oras.DefaultExtendedCopyOptions,
-		)
+		desc, err = oras.ExtendedCopy(ctx, src, srcRef.ReferenceOrDefault(), dst, dstRef.ReferenceOrDefault(), oras.DefaultExtendedCopyOptions)
 	} else {
-		desc, err = oras.Copy(ctx,
-			src,
-			srcRef.ReferenceOrDefault(),
-			dst,
-			dstRef.ReferenceOrDefault(),
-			oras.DefaultCopyOptions,
-		)
+		desc, err = oras.Copy(ctx, src, srcRef.ReferenceOrDefault(), dst, dstRef.ReferenceOrDefault(), oras.DefaultCopyOptions)
```

Comment:
Not a strong opinion, I'd like to create a struct wrapping up related params instead of passing too many params directly.